<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvFit</class>
 <widget class="QWidget" name="ConvFit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="MantidQt::CustomInterfaces::IDA::IndirectFitDataView" name="fitDataView" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="loPlotArea" stretch="1,1">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item>
          <widget class="MantidQt::CustomInterfaces::IDA::IndirectFitPropertyBrowser" name="fitPropertyBrowser">
           <widget class="QWidget" name="dockWidgetContents_2"/>
          </widget>
         </item>
         <item>
          <widget class="MantidQt::CustomInterfaces::IDA::IndirectFitPlotView" name="pvFitPlotView" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="loSpectrumView" stretch="0">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="MantidQt::CustomInterfaces::IDA::IndirectSpectrumSelectionView" name="svSpectrumView" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="gbRun">
            <property name="title">
             <string>Run</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>7</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pbRun">
               <property name="text">
                <string>Run</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="MantidQt::CustomInterfaces::IDA::IndirectFitOutputOptionsView" name="ovOutputOptionsView" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::CustomInterfaces::IDA::IndirectSpectrumSelectionView</class>
   <extends>QWidget</extends>
   <header>IndirectSpectrumSelectionView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::CustomInterfaces::IDA::IndirectFitPropertyBrowser</class>
   <extends>QDockWidget</extends>
   <header>IndirectFitPropertyBrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::CustomInterfaces::IDA::IndirectFitPlotView</class>
   <extends>QWidget</extends>
   <header>IndirectFitPlotView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::CustomInterfaces::IDA::IndirectFitDataView</class>
   <extends>QWidget</extends>
   <header>IndirectFitDataView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::CustomInterfaces::IDA::IndirectFitOutputOptionsView</class>
   <extends>QWidget</extends>
   <header>IndirectFitOutputOptionsView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
